rbi/sorbet/t.rbi

```diff
@@ -87,7 +87,9 @@ module T::Helpers
   def final!; end
   sig {void}
   def sealed!; end
+
   def mixes_in_class_methods(mod); end
+  def requires_ancestor(*mod); end
```

Comment:
Would you mind writing a signature for this? (Even if it's all untyped).

Reply:
It would be the only method in that file with a `sig`. Should I create a separate PR to also add signatures to the other `T::Helpers` methods?

Reply:
What do you mean? the `sealed!` and `final!` methods a few lines up also have sigs.

feel free to do this in a separate PR if you want.

Reply:
Oh sorry I was confusing this file with `helpers.rb` from sorbet-runtime: https://github.com/sorbet/sorbet/pull/3409/files#diff-f82649c582c88ac321fcaa37ca936da2R27-R43 do we want a signature in this file?

Reply:
Also, about adding a signature in `t.rbi`. It is the same problem than for `mixes_in_class_methods` and I believe it's why it doesn't have a signature either:

Since the checks for the `requires_ancestor` signature are done early in the `namer`, adding a signature would mean duplicating the errors during the `calls` checking.

For example:

```ruby
class A
  requires_ancestor "A"
end
```

Would return the error: 

```
error: `requires_ancestor` must only contain constant literals
```

As expected. But would also return:

```
error: Expected `Module` but found `NilClass` for argument `mod`
```

Because we discarded the node during the `namer` phase.

Reply:
gotcha; i'm fine to just have a comment there saying why there's no sig, for both of them.

Reply:
Good idea, I added a comment.